### ISSUE TITLE
Cant run on Rasberry pi Docker

### ISSUE THREAD

**Issue by HarveyTay** [alexemanuelol/rustplusplus#385]
When i try to run the docker on my rasberry pi i get this error along with an error about running from amd64 to arm64 and not specified

harvey@raspberrypi:~/rustplusplus $ sudo docker-compose up --build
Recreating rpp ... done
Attaching to rpp
rpp    | node: error while loading shared libraries: libdl.so.2: ELF load command address/offset not page-aligned
rpp exited with code 127

wondering of there is any workarounds or adjustments i can make for the code to run on this

**Comment by vaisest** [comment]
Sounds like the maintainer hasn't provided an ARM image. You could just build one yourself:

```bash
git clone https://github.com/alexemanuelol/rustplusplus.git && cd rustplusplus
docker build -t rustplusplus:latest .
```

Then change the image in your `docker-compose.yml` to rustplusplus:latest, and you should be able to run it in the same way

**Comment by PandeoF1** [comment]
> Sounds like the maintainer hasn't provided an ARM image. You could just build one yourself:
> 
> ```shell
> git clone https://github.com/alexemanuelol/rustplusplus.git && cd rustplusplus
> docker build -t rustplusplus:latest .
> ```
> 
> Then change the image in your `docker-compose.yml` to rustplusplus:latest, and you should be able to run it in the same way

Yup, it could be interesting to setup docker buildx arm, amd... to support multiple architecture
